### PR TITLE
action/debootstrap: make suite property mandatory for debootstrap action

### PR DESCRIPTION
**Pull request by vigneshraman**
Check if suite property is present for debootstarp action and contains
a string. If suite is not present, exit with an error.

Fixes: #238

Signed-off-by: Vignesh Raman <vignesh.raman@collabora.com>

### CODE REVIEW

**Comment by obbardc** [comment]
hmm, may be better to set a default suite (`main` for debian) and hope debootstrap complains for other distros ;-)

**Comment by vigneshraman** [comment]
> hmm, may be better to set a default suite (`main` for debian) and hope debootstrap complains for other distros ;-)

default mirror is http://deb.debian.org/debian
default component is main

So for suite can we use 'stable' as default ?

File: actions/debootstrap_action.go [hunk 1]
```diff
@@ -161,6 +161,10 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 	d.LogStart()
 	cmdline := []string{"debootstrap"}
 
+	if len(d.Suite) == 0 {
```
Comment (obbardc) [inline actions/debootstrap_action.go:164]: move to `Verify` function so that we can error out early

File: actions/debootstrap_action.go [hunk 2]
```diff
@@ -161,6 +161,10 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
...
+		return fmt.Errorf("'suite' is a mandatory property for Debootstrap Action")
```
Comment (obbardc) [inline actions/debootstrap_action.go:165]:
```suggestion
		return fmt.Errorf("suite property not specified")
```

**Review by obbardc — CHANGES_REQUESTED** [review]
two small nits

**Comment by vigneshraman** [comment]
@obbardc, have updated the commit.

**Review by obbardc — APPROVED** [review]

**Pull request merged**